sonar-xml-plugin/src/test/java/org/sonar/plugins/xml/checks/XmlSchemaCheckTest.java

```diff
@@ -176,7 +177,7 @@ public void fail_due_to_mixture_of_dtd_and_xsd_with_schema_provided() {
     parseAndCheck(xmlFile, createCheck(schema, null));
     
     assertLog("Unable to validate file .*entities\\.xml.*", true);
-    assertLog("Cause: .*nested\\.xml.*No such file or directory.*", true);
+    assertLog("Cause: .*nested\\.xml.*(No such file or directory|The system cannot find the file specified).*", true);
```

Comment:
@gjd6640 Could you make this change more explicit, e.g. create local vars for both OS.